packages/client/src/components/reports/scape/ScapeReportViewer.tsx

```diff
@@ -72,6 +71,24 @@ export default class ScapeReportViewer extends ConnectedComponent<Props, Stores,
       });
       throw e;
     }
+    this.setState({
+      scapeExecutionStatusResponse,
+      executionId
+    });
+  }
+
+  async componentDidMount(): Promise<void> {
+    const executionId = Optional.ofNullable(this.props.match.params.executionId).orElse('');
+    const self = this;
+
+    this.fetchScapeResponse(executionId);
+    const intervalId = setInterval(function() {
+      self.fetchScapeResponse(executionId);
```

Comment:
looks like `fetchScapeResponse` will be called twice if `scapeExecutionStatusResponse` is true initially.  Consider only calling in your interval function only if `scapeExecutionStatusResponse` is false.